pinot-controller/src/test/java/org/apache/pinot/controller/utils/SegmentMetadataMockUtils.java

```diff
@@ -81,15 +79,13 @@ public static SegmentMetadata mockSegmentMetadataWithPartitionInfo(String tableN
     if (columnName != null) {
       when(segmentMetadata.getColumnMetadataFor(columnName)).thenReturn(columnMetadata);
     }
-    when(segmentMetadata.getTableName()).thenReturn(tableName);
     when(segmentMetadata.getName()).thenReturn(segmentName);
     when(segmentMetadata.getCrc()).thenReturn("0");
     return segmentMetadata;
   }
 
   public static SegmentMetadata mockSegmentMetadataWithEndTimeInfo(String tableName, String segmentName, long endTime) {
     SegmentMetadata segmentMetadata = Mockito.mock(SegmentMetadata.class);
-    Mockito.when(segmentMetadata.getTableName()).thenReturn(tableName);
```

Comment:
this seems to be from segment metadata? (not zk)

Reply:
reverted